src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ReferralAmendmentDetails.kt

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+data class ReferralAmendmentDetails(
+  var values: List<String>
```

Comment:
Can you rename this complexityLevelIds or something similiar. this will help us in adding the other amends

Reply:
changed